tests/test_cli.py

```diff
@@ -101,11 +102,56 @@ def test_cli_exit_on_noinput_and_replay(mocker):
         "You can not use both replay and no_input or extra_context "
         "at the same time."
     )
+
     assert expected_error_msg in result.output
 
     mock_cookiecutter.assert_called_once_with(
         template_path,
         None,
         True,
-        replay=True
+        replay=True,
+        overwrite_if_exists=False
     )
+
+@pytest.mark.usefixtures('remove_fake_project_dir')
+def test_cli_exit_on_overwrite_if_exists_and_replay(mocker):
```

Comment:
I see. :no_mouth: 

The name is a bit misleading, I'm afraid. I thought it is supposed to test that **replay** and **overwrite** cause a the cli to exit without running **cookiecutter** (see ``test_cli_exit_on_noinput_and_replay`` which asserts for an error exit code).